crates/chat-cli/src/cli/chat/tool_manager.rs

```diff
@@ -2112,7 +2112,9 @@ mod tests {
         // Create mock prompt bundles
         let prompt = rmcp::model::Prompt {
             name: "test_prompt".to_string(),
+            title: Some("Test Prompt".to_string()),
             description: Some("Test description".to_string()),
+            icons: None,
```

Comment:
This change is not related to this PR, but I fixed it because a syntax error was occurring (test data modification).